pkgs/servers/clickhouse/default.nix

```diff
@@ -1,23 +1,23 @@
-{ lib, stdenv, fetchFromGitHub, cmake, libtool, llvm-bintools, ninja
+{ lib, llvmPackages_13, fetchFromGitHub, cmake, libtool, llvm-bintools, ninja
 , boost, brotli, capnproto, cctz, clang-unwrapped, double-conversion
 , icu, jemalloc, libcpuid, libxml2, lld, llvm, lz4, libmysqlclient, openssl, perl
 , poco, protobuf, python3, rapidjson, re2, rdkafka, readline, sparsehash, unixODBC
 , xxHash, zstd
 , nixosTests
 }:
 
-stdenv.mkDerivation rec {
+llvmPackages_13.stdenv.mkDerivation rec {
```

Comment:
Lets revert this and overwrite stdenv in top-level to make less changes.

Reply:
Just to confirm we're on the same page, you mean something like this?
https://github.com/NixOS/nixpkgs/blob/master/pkgs/servers/zookeeper/default.nix#L2-L6

Reply:
Glad that you asked.

I though about doing it like this:

https://www.github.com/NixOS/nixpkgs/blob/2f1e03cb2b4ce0d8823199f260e619bf4d19a6b5/pkgs/top-level/all-packages.nix#L992

``` 
afl = callPackage ../tools/security/afl {
  stdenv = clangStdenv;
};
```